src/charts/brush.js

```diff
@@ -109,9 +109,13 @@ export default function Brush (_container) {
       (d3.event.sourceEvent && d3.event.sourceEvent.type === "brush")) {
       return
     }
+    
     const extentClamped = getDataExtentUnderBrush()
-    cache.root.call(d3.event.target.move, extentClamped.map((d) => scales.xScale(d)))
-    dispatcher.call("brushMove", this, getDataExtentUnderBrush(), config)
+
+    if (extentClamped) {
```

Comment:
Just a note, but you could even use `if (extendIsValid(extentClamped)) {` which would also catch invalid extents.

Reply:
Good call, fixed.